python/CMakeLists.txt

```diff
@@ -16,9 +16,9 @@ cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
 
 set(rmm_version 22.06.00)
 
-set(CYTHON_FLAGS
-    "--directive binding=True,embedsignature=True,always_allow_keywords=True"
-    CACHE STRING "The directives for Cython compilation.")
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.06/RAPIDS.cmake
```

Comment:
This probably requires an update to https://github.com/rapidsai/rmm/blob/branch-22.06/ci/release/update-version.sh

Reply:
Good catch.